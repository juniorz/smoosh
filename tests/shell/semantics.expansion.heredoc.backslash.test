cat <<EOF
an escaped \\[bracket]
should \\ work just fine
EOF