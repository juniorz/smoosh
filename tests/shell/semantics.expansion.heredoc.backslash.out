an escaped \[bracket]
should \ work just fine
exit $?
